lib/ae_mdw/auction_bids.ex

```diff
@@ -113,16 +114,21 @@ defmodule AeMdw.AuctionBids do
          Model.auction_bid(
            index: plain_name,
            block_index_txi_idx: {block_index, _txi_idx},
-           expire_height: expire_height,
-           bids: [last_bid | _rest_bids]
+           expire_height: expire_height
          ),
          last_gen,
          last_micro_time,
          _opts
        ) do
+    last_bid_txi =
+      state
+      |> Name.stream_nested_resource(Model.AuctionBidClaim, plain_name)
+      |> Enum.at(0)
```

Comment:
It's already readable now but there are 3 uses of stream and `Enum.at(0)`. Maybe we could have a new function that picks the first?

Reply:
I needed to add 2 new functions for both stream_nested_resource/3 and /4. Or add a single function that only did `Enum.at(stream, 0)`. I don't think it's worth it either way

Reply:
There is one that is doing it already, the [last_update](https://github.com/aeternity/ae_mdw/blob/1333d624a9a23309fb5ae27a453914199abeb175/lib/ae_mdw/db/name.ex#L149). This [ownership case](https://github.com/aeternity/ae_mdw/blob/1333d624a9a23309fb5ae27a453914199abeb175/lib/ae_mdw/db/name.ex#L169) would be a little bit more readable.

Reply:
You're right, done